### DISCUSSION TITLE
Problems in Prior Austenite Grain Reconstruction

### DISCUSSION THREAD

**Discussion by tuliocn** [mtex-toolbox/mtex#790] [Ask Anything]
Hello everyone.

I am working on prior austenite grain reconstruction of one sample of HSLA steel that was austenitizing at 1230°C for 15 minutes, and after that was quenching on dilatometer equipament. I am trying to reconstruct the PAG of this sample from EBSD map, but the results it was not good. 

I am using the MTEX 5.6.0, and using the code described on MTEX site (https://mtex-toolbox.github.io/MaParentGrainReconstruction.html). 

I have tried to change some values in the job.grains fit and job.grains.clusterSize to improve the results, but I didn't have good results yet. (job.revert(job.grains.fit > 5*degree | job.grains.clusterSize < 15)

Somebody can help me to understand what can I changed to obtain good results in this case? I am sending attached the results of the PAG of this sample and EBSD file of the sample. 

Best regards,

[pre deformada mapa 1.zip](https://github.com/mtex-toolbox/mtex/files/6006961/pre.deformada.mapa.1.zip)

![12](https://user-images.githubusercontent.com/79291369/108442542-00abed80-7236-11eb-9403-0a8e0f75d5a9.jpg)

**Comment by nyyssont** [comment]
Hello Tulio,

you have too many phases in your phase list for the parentGrainReconstructor to guess which is the prior and which is the product phase. When you enter:

```
ebsd.mineralList
```

I think you'll find you have cementite listed in the phase list, in addition to ferrite and austenite. In this case, you need to supply an example of the orientation relationship when you are creating the parentGrainReconstructor.

I suggest trying the following:

```
cs_bcc = ebsd('Iron bcc').CS;
cs_aus = ebsd.CSList{4};

KS = orientation.map(Miller(1,1,1,cs_aus),Miller(0,1,1,cs_bcc),...
      Miller(-1,0,1,cs_aus),Miller(-1,-1,1,cs_bcc));

job = parentGrainReconstructor(ebsd,grains,KS) % define job
```

And proceeding from there.

-Tuomo
